layouts/partials/header.html

```diff
@@ -32,45 +33,37 @@
                 </ul>
             </li>
         </ul>
-        <!-- <a href="/docs/home" class="button" id="viewDocs" data-auto-burger-exclude>View Documentation</a> -->
-        <a href="/docs/tutorials/kubernetes-basics/" class="button" id="tryKubernetes" data-auto-burger-exclude>Try Kubernetes</a>
+        {{ with .Site.GetPage "section" "docs/tutorials/kubernetes-basics" }}
+        <a href="{{ .RelPermalink }}" class="button" id="tryKubernetes" data-auto-burger-exclude>{{ .LinkTitle }}</a>
+        {{ end }}
+
         <button id="hamburger" onclick="kub.toggleMenu()" data-auto-burger-exclude><div></div></button>
     </div>
 
     <nav id="mainNav">
         <main data-auto-burger="primary">
-        <div class="nav-box">
-            <h3><a href="/docs/tutorials/stateless-application/hello-minikube/">Get Started</a></h3>
-            <p>Ready to get your hands dirty? Build a simple Kubernetes cluster that runs "Hello World" for Node.js.</p>
-        </div>
-        <div class="nav-box">
-            <h3><a href="/docs/home/">Documentation</a></h3>
-            <p>Learn how to use Kubernetes with the use of walkthroughs, samples, and reference documentation. You can even <a href="/editdocs/" data-auto-burger-exclude>help contribute to the docs</a>!</p>
-        </div>
-        <div class="nav-box">
-            <h3><a href="/community/">Community</a></h3>
-            <p>If you need help, you can connect with other Kubernetes users and the Kubernetes authors, attend community events, and watch video presentations from around the web.</p>
-        </div>
-        <div class="nav-box">
-            <h3><a href="/blog/">Blog</a></h3>
-            <p>Read the latest news for Kubernetes and the containers space in general, and get technical how-tos hot off the presses.</p>
+         {{ range .Site.Menus.main }}
+           <div class="nav-box">
+            <h3><a href="{{ .URL }}">{{ .Title }}</a></h3>
+           {{ .Post }}
         </div>
+         {{ end }}
         </main>
         <main data-auto-burger="primary">
         <div class="left">
-            <h5 class="github-invite">Interested in hacking on the core Kubernetes code base?</h5>
-            <a href="https://github.com/kubernetes/kubernetes" class="button" data-auto-burger-exclude>View On Github</a>
+            <h5 class="github-invite">{{ T "main_github_invite" }}</h5>
```

Comment:
@bep Cool implementation. Are these handlebars expressions?

Reply:
It isn't a shortcode, it's a template function: https://gohugo.io/functions/i18n/